src/components/AutoSuggest/AutoSuggest.tsx

```diff
@@ -387,7 +386,7 @@ const AutoSuggest = ({
             variant="blank"
           />
         ) : (
-          <ClearInput
+          <InlineButton
```

Comment:
Maybe out of scope, but: this renders a button that doesn't really do anything other than setting the focus on the input. Shouldn't this just render an icon? Similar to what they do [here](https://onderzoek.amsterdam.nl/zoek)

Reply:
True. But then the input is not focussed anymore. Will leave it for now.